tigerpath/majors_and_certificates/majors/NES_2018.json

```
@@ -73,20 +73,157 @@
+                    "NES 315",
+                    "NES 326",
+                    "NES 327",
+                    "NES 338",
+                    "NES 362",
+                    "NES 365",
+                    "NES 373",
+                    "NES 374",
+                    "NES 377",
+                    "NES 385",
+                    "NES 391",
+                    "NES 394",
+                    "NES 395",
+                    "NES 433",
+                    "NES 438"
+                  ]
+                }
+              ]
             },
             {
               "name": "Regions",
-              "max_counted": null,
-              "min_needed": null,
-              "explanation": "<i>Editor's Note: This requirement is too broad or requires course approval from the concentrator's Director of Undergraduate Studies, so TigerPath does not check it.</i>\n\nAt least two regionally specific courses. Each must focus on one of the following six sub-regions of the Near East. They may not focus on the same region: Egypt, North Africa, and Andalusia; The Levant, Iraq, and the Arabian peninsula; Ottoman empire/Turkey; Iran; Muslim South and Central Asia; Diasporic communities.",
-              "no_req": null
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "At least two regionally specific courses. Each must focus on one of the following six sub-regions of the Near East. They may not focus on the same region: Egypt, North Africa, and Andalusia; The Levant, Iraq, and the Arabian peninsula; Ottoman empire/Turkey; Iran; Muslim South and Central Asia; Diasporic communities.",
+              "req_list": [
+                {
+                  "name": "Egypt, North Africa, and Andalusia",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "<i>Editor's Note: This does not represent a comprehensive list of courses that fulfill this requirement as courses change from semester to semester.</i>",
+                  "course_list": [
+                    "NES 251",
+                    "NES 269",
+                    "NES 315",
+                    "NES 369",
+                    "NES 390",
+                    "NES 394",
+                    "NES 395"
+                  ]
+                },
+                {
+                  "name": "The Levant, Iraq, and the Arabian Peninsula",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "<i>Editor's Note: This does not represent a comprehensive list of courses that fulfill this requirement as courses change from semester to semester.</i>",
+                  "course_list": [
+                    "NES 221",
+                    "NES 269",
+                    "NES 326",
+                    "NES 338",
+                    "NES 373"
+                  ]
+                },
+                {
+                  "name": "Ottoman Empire/Turkey",

```

The `/` is causing it's search to fail. It's a known issue, unfortunately, but until we take the time to fix it, we have to avoid `/`'s.

One option is something like this
```suggestion
                  "name": "Ottoman Empire, Turkey",
```
But whatever alternative name you find appropriate is fine.